### PR TITLE
Better territories

### PR DESCRIPTION
**Pull request by davidbgk**
This is a 3 steps process:

First the user is not logged in

capture d ecran 2016-03-22 a 14 42 27

Then we encourage him to publish datasets for that zone

capture d ecran 2016-03-22 a 14 43 07

Finally ask him to claim that town to be referenced first

capture d ecran 2016-03-22 a 14 43 36

Labels and styles must be updated but it's more a discussion on the workflow ATM.

### CODE REVIEW

**Comment by davidbgk** [comment]
Don't forget to review the gouvfr counterpart too for tests: https://github.com/etalab/udata-gouvfr/pull/97

File: udata/core/organization/commands.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+from udata.commands import manager
```
Comment (noirbizarre) [inline udata/core/organization/commands.py]: better use a submanager for namespacing on organization.
See: https://github.com/etalab/udata/blob/master/udata/harvest/commands.py#L10-L17

File: udata/core/organization/commands.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+from udata.models import GeoZone, Organization
+
+log = logging.getLogger(__name__)
+
+
+@manager.command
+def attach_zone(insee_code, organization_id_or_slug, level='fr/town'):
```
Comment (noirbizarre) [inline udata/core/organization/commands.py]: Optionnal: limit level values to granularities, with `@manager.option(choices=...)` (or enum)

File: udata/core/user/models.py [hunk 3]
```diff
@@ -169,6 +169,12 @@ def generate_api_key(self):
     def clear_api_key(self):
         self.apikey = None
 
+    def can_modify(self, datasets):
```
Comment (noirbizarre) [inline udata/core/user/models.py]: can work on reuses too, so maybe generalize `dataset` to `model`

Comment (noirbizarre) [inline udata/core/user/models.py]: In fact, there is the dataset permission module that centralize these rules, better use it.

**Comment by noirbizarre** [comment]
r+wc

File: udata/core/organization/commands.py (hunk 2 above)
Comment (davidbgk) [inline udata/core/organization/commands.py]: It introduces a bit of complexity, let's do that when we'll introduce more levels.

**Comment by noirbizarre** [comment]
r+

**Pull request merged**